bessctl/commands.py

```diff
@@ -269,6 +269,11 @@ def get_var_attrs(cli, var_token, partial_word):
             var_desc = 'configuration filename'
             var_candidates = complete_filename(partial_word)
 
+        elif var_token == 'LIB_FILE':
+            var_type = 'filename'
+            var_desc = 'library filename'
+            var_candidates = complete_filename(partial_word)
```

Comment:
How about something like:
```
var_candidates = complete_filename(partial_word, suffix='.so', skip_suffix=True)
```
so that only *.so files can be considered for autocompletion? (`skip_suffix` is not implemented yet btw)